plugin.video.aktualnetv/resources/language/resource.language.cs_cz/strings.po

```diff
@@ -28,63 +28,3 @@ msgstr "Aktuálně.cz — kompletní zpravodajství, zprávy z domova i ze svět
 msgctxt "Addon Disclaimer"
 msgid "This is an unofficial addon and is in no way affiliated with publisher Economia, a. s. It was created and is maintained by an independent developer. It is provided for reference only and may contain errors."
 msgstr "Toto je neoficiální doplněk a není, žel, žádným způsobem spojen s vydavatelstvím Economia, a. s. Byl vytvořen a je udržovan nezávislým vývojářem. Doplněk je poskytnut pouze jako ukázka a může obsahovat chyby."
-
-msgctxt "#30000"
-msgid "Aktualne TV"
-msgstr "Aktuálně TV"
```

Comment:
I think removing all strings wasn't your intention.

Reply:
OMG, you are right, my mistake. Thank you for this check.